Glossary (#490)
nfidd/nfidd: about/glossary.qmd

---
title: "Glossary of key concepts"
---

## Session 1: Statistical concepts

**Probability density function**

**Probability mass function**

**Sampling the posterior**

### Session 2

**Primary and secondary events**: when one epidemiological event (the primary event) directly causes another epidemiological event (the secondary event).
For example, infection and symptom onset; infection in one person causing infection in another.
A focus of this course is on understanding the time between these pairs of events.
Key pairs of events include: 
- The incubation period: infection to symptom onset 
- Symptom onset to hospitalisation 
- The generation time: infection in one person causing infection in another

### Session 3

**Censoring**: knowing that an event occurred at some time, but not specifically when.

> In this course, we mostly consider double (or interval) censored events: we know the event occurred at some time within a time interval, such as at any time on a certain date.

Reviewer: ```suggestion
In this course, we mostly consider double (interval) censored events: we know that both events occurred at some time within a time interval, such as at any time on a certain date.
```

They are double interval censored.

> Events might also be right censored (we only know the earliest time it could have happened) or left censored (we only know the latest time it could have occurred).

Reviewer: I think these are the wrong way around? I think if something is right censored we know events up until a latest time and then beyond that we know they happened but not the value.

> **Right truncation**: when we only have incomplete information about events because not enough time has passed to observe all of them.

Reviewer: It would be nice to standardise the here as truncation has the same flavours as censoring with the added flavour of we also don't k ow if events occurred. I think the Charniga paper has some defs we can lift has these things are tricky to talk about.

Reviewer: By standardise I mean introduce censoring and truncation then types I.e right left interval and the the idea of double which is just where both events in a delay have some censoring

This is a problem if we try to use incomplete secondary event data (such as recent symptom onsets) to infer the pattern of primary events - our estimates will be biased because we are missing some of the data.

### Session 4

**Convolution**

**Discretisation**: when we want to use a continuous probability distribution to model a discrete (count) variable, we need to transform the probability distribution from continuous to discrete.

Moving back and forth between continuous and discrete / simulation and estimation

### Session 5

**Reproduction number**: the average number of secondary infections caused by a single infectious individual and can in time and space as a function of differences in population level susceptibility, changes in behaviour, policy, seasonality etc.

> **Renewal equation**: relates the number of infected people in the population; the current value of the reproduction number; and a delay distribution of the so-called generation time

Reviewer: ```suggestion
**Renewal equation**: relates the number of infected people in the population; the current value of the reproduction number; and a delay distribution of the generation time
```

> **Random walk**: random noise along a given trajectory. In this course, we use a **geometric random walk**, where we add this random noise the logarithm of the reproduction number.

Reviewer: ```suggestion
**Random walk**: random noise along a given trajectory. In this course, we use a **geometric random walk**, where we add this random noise to the logarithm.
```
